components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java

```diff
@@ -1118,6 +1120,93 @@ public OAuthRevocationResponseDTO revokeAuthzForAppsByResourceOwner(
         return new OAuthRevocationResponseDTO();
     }
 
+    /**
+     * Revoke issued tokens for the application.
+     *
+     * @param application {@link OAuthAppRevocationRequestDTO}
+     * @return revokeRespDTO {@link OAuthAppRevocationRequestDTO}
+     * @throws IdentityOAuthAdminException Error while revoking the issued tokens.
+     */
+    public OAuthRevocationResponseDTO revokeIssuedTokensByApplication(OAuthAppRevocationRequestDTO application)
+            throws IdentityOAuthAdminException {
+
+        triggerPreApplicationTokenRevokeListeners(application);
+
+        String consumerKey = application.getConsumerKey();
+        String applicationName = application.getApplicationName();
+        if (consumerKey != null && applicationName != null) {
```

Comment:
let's use StringUtils.isNotBlank

Reply:
Addressed in [40fde89](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1627/commits/40fde89cb099493c8f2a302b34b1141d9aea3e8d)